### ISSUE TITLE
Setup does not detect any controllers

### ISSUE THREAD

**Issue by Kizoky** [chocolate-doom/chocolate-doom#1169]
### Background

Version of Chocolate Doom: chocolate-doom-20190601-win32

Operating System and version: Windows 10 Home (1809 17763.503)

Game: (Doom/Heretic/Hexen/Strife/other)
none

Any loaded WADs and mods (please include full command line):
none

### Bug description

Observed behavior:
When trying to configure a controller, the setup does not detect my controller (it's a X360 controller)

Expected behavior:
It should be able to detect my controller

This is a problem that seems to be within the SDL, since copying the SDL files from the 3.0.0 Chocolate Doom release, the setup is able to see my controller

**Comment by fabiangreffrath** [comment]
There is `SDL_INIT_GAMECONTROLLER`, maybe we want to use that?

**Comment by fragglet** [comment]
No, as I recall, SDL_INIT_GAMECONTROLLER is just a layer on top of the normal SDL_INIT_JOYSTICK subsystem.

The fact that copying the SDL files from 3.0.0 fixes the problem does seem to point to this being an SDL bug. I wonder if there's anything in their bugtracker about it.

As mentioned on #1171, maybe we should just roll back to an older SDL version.

**Comment by fabiangreffrath** [comment]
> As mentioned on #1171, maybe we should just roll back to an older SDL version.

I am all for it, as long as MP3 playback remains intact. Speaking of which, how about adding fluidsynth MIDI rendering?

**Comment by JNechaevsky** [comment]
Have a suspicion that it's not a SDL 2.0.9 issue - I have two pretty simple USB joysticks, both of them are not detected by latest dev builds of Chocolate and Crispy, but working fine in RusDoom despite of I have updated SDL to 2.0.9. 

Probably, something goes wrong in the joystick code itself, I'm still using old code from Chocolate 3.0 beta in RD.

**Comment by filevans** [comment]
is this being fixed? it says no controller detected on my xbox 360 usb controller, windows 7 64bit
